extension/popup/popupController.jsx

```diff
@@ -396,10 +399,27 @@ export const PopupController = function() {
         type: "addTelemetry",
         properties: { transcriptionConfidence: json.data[0].confidence },
       });
-      browser.runtime.sendMessage({
+
+      const intentComplete = await browser.runtime.sendMessage({
         type: "runIntent",
         text: json.data[0].text,
       });
+
+      if (intentComplete) {
+        const runFollowUp = await browser.runtime.sendMessage({
+          type: "getWaitingOnFollowUp",
+        });
+
+        waitForFollowUp = runFollowUp;
+        if (runFollowUp) {
+          recorder.startRecording();
+          runFollowUpTimerId = setTimeout(async () => {
+            await browser.runtime.sendMessage({
+              type: "stopListeningForFollowUp",
+            });
+          }, 5000);
```

Comment:
Is the idea here that after 5 seconds it stops listening for a followup?

The recorder also has a silence timeout that might be the right thing to use here.

Reply:
This would complete the intent and close the pop up window, since it was being held open. I could move this to the context as well, but thought the user might want to set the timer in their settings.